src/pages/LoginForm.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from "react";
+import {
+  Flex,
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  CircularProgress,
+  Text,
+  Link,
+} from "@chakra-ui/core";
+
+import ErrorMessage from "../components/ErrorMessage";
+
+const LoginForm = () => {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    isLoading: false,
+    error: "",
```

Comment:
Also can you set error as null instead of empty string. I know it will evaluate to false in javascript but null seems much more clearer.